docs.html

```diff
@@ -943,27 +915,21 @@ <h1>Modal Title</h1>
                 </header>
                 <main class="modal__body">
                   <p>
-                    Lorem ipsum dolor sit amet consectetur adipisicing elit.
-                    Omnis blanditiis, explicabo deleniti rerum.
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Omnis blanditiis,
+                    explicabo deleniti rerum.
                   </p>
                 </main>
                 <footer class="modal__footer" id="modal-footer">
-                  <button class="btn btn--primary modal__footer--btn">
-                    ACTION 1
-                  </button>
-                  <button class="btn btn--outline modal__footer--btn">
-                    ACTION 2
-                  </button>
+                  <button class="btn btn--primary modal__footer--btn">ACTION 1</button>
+                  <button class="btn btn--outline modal__footer--btn">ACTION 2</button>
                 </footer>
               </section>
             </div>
             <script>
               const modalBtn = document.getElementById("modal-btn");
               const modalContainer = document.getElementById("modal-container");
               const modalElem = document.getElementById("modal-el");
-              const modalFooterButtons = document.querySelectorAll(
-                ".modal__footer--btn"
-              );
+              const modalFooterButtons = document.querySelectorAll(".modal__footer--btn");
 
               modalBtn.addEventListener("click", () => {
```

Comment:
suggestion: create a different js file for the logic, keep the structure and logic separate in your code.